debuginfo/src/dwarf.rs

```diff
@@ -483,11 +476,8 @@ impl<'d, 'a> DwarfUnit<'d, 'a> {
             return Ok(tuple);
         }
 
-        // to go by the logic in dwarf2read a low_pc of 0 can indicate an
-        // eliminated duplicate when the GNU linker is used.
-        // TODO: *technically* there could be a relocatable section placed at VA 0
         let low_pc = match low_pc {
-            Some(low_pc) if low_pc != 0 => low_pc,
+            Some(low_pc) => low_pc,
```

Comment:
@nnethercote I'm afraid we'll need to make this check conditional, as you suggested. See https://github.com/getsentry/symbolic/pull/173